### DISCUSSION TITLE
Receber em AUD (dólar australiano) sem dupla conversão

### DISCUSSION THREAD

**Discussion by raulfragoso** [DyegoCosta/trabalhando-remoto#183] [General]
Pessoal, já faz anos que presto serviço para uma empresa na Austrália e recebo como pessoa física pelo Banco Inter -  pretendo mudar esse ano para PJ, visando uma diminuição tributária.

Um dos problemas que enfrento é a dupla conversão de AUD para USD e USD para BRL. Isso se dá pois o correspondente bancário do Banco Inter (e pelo que sei qualquer outro banco brasileiro) exige que o recebimento seja feito em USD, portanto eu já perco com o spread duas vezes: primeiro pela taxa de conversão utilizada pelo banco australiano na conversão AUD -> USD, depois pela conversão de USD -> BRL no fechamento do câmbio com o banco no Brasil.

Alguém aqui já teve o mesmo problema? Há alguma maneira de receber diretamente de AUD para BRL, evitando assim a dupla conversão com perda por spread?

Muito obrigado!

**Comment by diego-silva** [comment]
Como PF da pra usar a Borderless Account do Transferwise, mas infelizmente não aceitam PJ ainda. Como PJ a melhor opção que conheço é o Payoneer…
Ambos (Transferwise e Payoneer) te dariam um número de conta local Australiana, depois quando tu quiser tu faz a conversão para reais (por eles) e entra como um TED na tua conta BR.
Pelo Payoneer tem um spread de 2% na hora de conveter, o Transferwise é um pouco menos (mas não sei quanto exatamente).

Se quiser um referral link pra algum deles, ambos ganhamos alguma coisa (TW acho que é 1 transferencia sem tarifas e Payoneer é 25 doláres para cada)

**Comment by raulfragoso** [comment]
Muito obrigado pelas informações. Acho que vou pelo Payoneer mesmo, a empresa estará aberta até o fim da semana, então já posso dar início ao processo de cadastro e abertura de contas. E por falar em conta, alguma recomendação para PJ nos bancos digitais? Tenderei a ir pro Banco Inter pela boa experiência como PF, mas aceito sugestões.
Quanto ao referral link, pode mandar :) 

Valeu!

**Comment by diego-silva** [comment]
Para conta PJ no BR estou usando o Banco Inter mesmo, por enquanto está tranquilo mas só uso para receber a TED da Payoneer e enviar pra conta PF, então não faço nada de 'especial' com ela…

Segue o link do Payoneer: 
https://share.payoneer.com/nav/ZZKzCiAsV2Bp1G_a8B9FZIBNCZ2NP0pLVm9eTJC3fuSq5OkDY-03nYvxnpH27dMoRbZq51INX_ffEH4KuN-xEw2
PS.: no Payoneer tem que abrir como PJ mesmo, pq senão depois eles só fazem TED pra conta PF.

**Comment by raulfragoso** [comment]
Atualizando: a empresa já foi aberta e registrada em todos os órgãos necessários. Em seguida abri duas contas PJ, uma no Banco Inter e outra no BS2 (tive uma excelente experiência recentemente com a conta internacional e debit card do BS2 numa viagem a Londres), em ambos os bancos o processo foi tranquilo, levou 2 dias úteis para concluir a abertura das contas, e tudo de maneira digital. Também já criei a conta no Payoneer, inclusive a conta local na Australia, e tudo já está aprovado para começar receber transferências.

No começo de março irei receber a primeira transferência da empresa e atualizarei aqui novamente. Mais uma vez agradeço ao @diego-silva pela ajuda!

**Comment by raulfragoso** [comment]
Acabei de receber a primeira transferência da empresa local na conta da Austrália pelo Payoneer, mas ainda consta como 'Pending'. Havia um questionário pendente de resposta para saber mais sobre minha 'line of business', já preenchi e enviei, espero que até amanhã libere para withdraw na conta do Brasil. Atualizo aqui depois.

**Comment by clmadrazo** [comment]
Recomendo que não deixe à vontade deles liberar e entre em contato com eles via chat para perguntar o motivo da demora. Aí eles vão dizer que ainda está no prazo de análise e no dia seguinte ou no mesmo dia vão liberar. Do contrário vai ter que aguardar até o prazo de 3 dias esgotar. Já passei por isso duas vezes. Fica a dica ;)

**Comment by raulfragoso** [comment]
Nem precisei entrar em contato com eles, agora pouco chegou o email do Payneer informando que os fundos estavam disponíveis.
Fiz o withdraw para a conta PJ no banco BS2, e o Payoneer corretamente cobrou a taxa de 2% sobre a cotação oficial do AUD, pagando R$ 2,94 para R$ 3,00 da cotação oficial deste momento (que aliás é a máxima dos últimos 18 meses \o/)
Acredito que isso encerra o ciclo, e que agora todo mês poderei receber tranquilamente usando o Payoneer.
Obrigado a todos que comentaram e ajudaram!

**Comment by diego-silva** [comment]
Que bom que deu tudo certo @raulfragoso 🎉  deve ter sido bem melhor que pagar o teu spread duplo 😂.

Eu tenho normalmente recebido no mesmo dia que ela aparece em pendente, mas teve uma vez que já estava demorando mais do que 30 horas (que eu notei que estava pending), entrei no chat e mandaram esperar, que é normal levar até 3 dias.
Passou 3 dias e ainda não tinha entrado, entrei no chat e falaram que tinha que esperar mais 3 dias, aí perguntei se estavam de brincadeira comigo pq ja tinham disso isso a 3 dias atrás. A pessoa do chat ficou um tempo sem falar nada (uns 5 minutos) e depois disse 'já consegui, pode verificar na sua conta'.

Então no geral a experiência tem sido bem boa, só mesmo essa vez que demorou mais, mas ainda assim foi tranquilo… 
O único problema do chat é o tempo de espera, fiquei mais de 1h na 'fila'.

**Comment by raulfragoso** [comment]
Sim @diego-silva  pensa num cabra que tá feliz 😄  Na média, vou economizar (ou deixar de perder) 6% que tinha com o double spread 😃Junte a isso uma diminuição de ~ 7% do IR por conta da empresa (versus PF como era antes) e já sou um cara ainda mais feliz 🤣 

Bom saber que o atendimento deles funciona, ainda que demore. Mas como estamos no Brasil e já calejados de ligar no call center da Vivo, Sky, Correios, etc isso é fichinha 😆 

Valeu!

**Comment by vrbrito** [comment]
@raulfragoso Como tem sido sua experiência com o BS2? Estou pensando em começar a receber em dólares diretamente do Upwork para minha conta internacional do BS2 e então fazer a conversão para reais. Você acha que pelo Payoneer sairia ainda mais barato que desta maneira?

**Comment by raulfragoso** [comment]
@vrbrito Por enquanto a experiência com o BS2 tem sido muito boa, mas ainda não utilizei o BS2 para receber ordens do exterior. Para essa finalidade eu recomendo o Banco Inter, que utilizei para receber como pessoa física até o mês passado (agora migrei para PJ + Payoneer, conforme a saga acima).
O Inter usa um spread bem baixo, cerca de 1%, e não há taxas adicionais. Também gostei do atendimento, toda comunicação era por email, e tudo que me pediram no início foi o contrato de prestação de serviços (Software Development Agreement). Daí para cada ordem recebida eu fechava o câmbio pelo app e só tinha que enviar o respectivo invoice da ordem de pagamento recebida.

**Comment by vrbrito** [comment]
@raulfragoso Bem legal esse spread do banco inter, obrigado pela informação! Só não sei que documento vou fornecer a eles, já que utilizo o Upwork para gerenciar os trabalhos, mas vou conversar com o suporte deles para ver isso.

**Comment by 123dma** [comment]
Opa, opero com o banco inter, na verdade eles não te spread fixo. Varia de
acordo com os critérios deles acho q de 1 a 3%. Em um mês tive uma
diferença de 12 centavos em relação a cotação de momento do dólar. É
competitivo, as vezes vale a pena outras vezes não. mas não é 1% não. Opero
pj

Em dom, 15 de mar de 2020 14:30, Vitor Ramalho <notifications@github.com>
escreveu:

**Comment by 123dma** [comment]
Corrigindo pessoal, fiquei nos email aqui. spread 1 a 2%. Não foi 13 mas 9
centavos.


Em dom, 15 de mar de 2020 15:07, Diego ma <diegomamorim@gmail.com> escreveu:

**Comment by lucas-barros** [comment]
Alguem tem experiência com o Remessa Online? Comparando com o banco inter, qual oferece a melhor experiência?

**Comment by 123dma** [comment]
Remessa online é fixo 1%. Fiz no último mês com eles, achei  meio demorado,
mas dei o desconto pois a remessa veio na segunda de carnaval, consegui
finalizar na quinta.

Em dom, 15 de mar de 2020 16:47, Lucas Araujo <notifications@github.com>
escreveu:

**Comment by chapeleiro775** [comment]
Estou tentando receber também da Austrália minha conta é inter física mesmo e me aparecem 2 códigos swift para realizar a transferência, mas, ocorre que na opção do pagador só dá pra inserir um só codigo, alguém está passando pelo mesmo problema?

**Comment by raulfragoso** [comment]
> Estou tentando receber também da Austrália minha conta é inter física mesmo e me aparecem 2 códigos swift para realizar a transferência, mas, ocorre que na opção do pagador só dá pra inserir um só codigo, alguém está passando pelo mesmo problema?

É provável que seja o código IBAN, eu recebia por ele também quando usava a conta PF a mais de 2 anos (hoje é PJ com Payonner, bem mais prático). Se é o IBAN, você pode calcula-lo utilizando essa ferramenta, basta selecionar país, banco, agência e conta: https://www.iban.com/calculate-iban
